CSharp/Library/Microsoft.Bot.Builder/Dialogs/IDialogContext.cs

```diff
@@ -141,6 +141,7 @@ public static partial class Extensions
         {
             var message = botToUser.MakeMessage();
             message.Text = text;
+            message.InputHint = InputHints.ExpectingInput;

```

Comment:
I agree that in the case of multiple messages on the stack we wouldn't want to do this. Let's make it so it only happens in the case where there's a single message on the stack. Can you help with that?